packages/account/src/Components/currency-selector/radio-button-group.tsx

```diff
@@ -36,7 +46,7 @@ const RadioButtonGroup = ({
             )}
             <div
                 className={classNames('currency-list__items', {
-                    'currency-list__items__center': item_count < 4,
+                    'currency-list__items__center': item_count && item_count < 4,
```

Comment:
```suggestion
                    'currency-list__items__center': item_count < 4,
```
it's either less, or greater than 4, so if it's a falsy value, the expression will give us false anyway 😊

Reply:
Will fix this